gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -156,24 +190,39 @@ private int benchmarkRead(FileSystem fs, Map<String, String> args) {
     warmup(
         args,
         () ->
-            benchmarkRead(
-                fs, testFile, /* readSize= */ 1024, /* numReads= */ 1, /* numThreads= */ 2));
+            benchmarkFileIO(
+                fs,
+                testFile,
+                true,
+                /* readSize= */ 1024,
+                /* numReads= */ 1,
+                /* numThreads= */ 2,
+                0));
```

Comment:
Please add comment on the argument

Reply:
replaced boolean with enum. This is more descriptive